pkg/cmd/util/sibling.go

```diff
@@ -26,5 +34,21 @@ func SiblingCommand(cmd *cobra.Command, name string) string {
 	command = append(command, name)
 	glog.V(4).Infof("The sibling command is: %s", strings.Join(command, " "))
 
-	return strings.Join(command, " ")
+	return command
+}
+
+// EscapeCommand wraps each element of the command in double quotes
+// if it contains any of the following: single quote, double quote, backslash, or space
+// It also replaces every pre-existing quoting character in the element with a pair.
+// Examples: ' -> '', " -> ""
+func EscapeCommand(command []string) []string {
```

Comment:
This escaping style is pretty rsync-specific so should the function should be named accordingly and be put in the appropriate rsync-related package